fedelemflowlist/input/GroupsFlowablePrimaryContexts.csv

```diff
@@ -492,4 +492,16 @@ Tetrabromotrifluoropropanes,Emission,Air
 Tribromotetrafluoropropanes,Emission,Air
 Dibromopentafluoropropanes,Emission,Air
 Hexabromofluoropropanes,Emission,Air
-Propylbenzene isomers,Emission,Air
\ No newline at end of file
+Propylbenzene isomers,Emission,Air
+Adsorbable organic halides,Emission,Air
+"Hydrocarbons, chlorinated",Emission,Air
+"Hydrocarbons, halogenated
```

Comment:
@ashleyedelen lines 498 and 499 have some issues, looks like 498 is missing some columns. Opening it in excel helps